packages/server/modules/core/tests/integration/user-emails.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { before } from 'mocha'
+import { createUser } from '../../services/users'
+import { beforeEachContext } from '@/test/hooks'
+import { expect } from 'chai'
+import { getUserByEmail } from '../../repositories/users'
+import { UserRecord } from '../../helpers/types'
+
+describe('Core @user-emails', () => {
+  before(async () => {
+    await beforeEachContext()
+  })
+  describe('getUserByEmail', () => {
+    it('should return null if user does not exist', async () => {
+      expect(await getUserByEmail('test@example.org')).to.be.null
+    })
+
+    it('should return user if user-email does not exist', async () => {
```

Comment:
actually no, I created this test to ensure the function still return the user has no user-email because until we will apply the migration to remove the user.email field we want this to work

Reply:
oh shit, you're right, I am an idiot :joy:

Reply:
this test was not supposed to be pushed here, I added a TODO to fix it in the next PR